README.md

```diff
@@ -1798,6 +1798,7 @@ _Related: [Task management & To-do lists](#task-management--to-do-lists), [Softw
 - [osTicket](https://osticket.com/) - Manage, organize and archive all your support requests and responses in one place. ([Source Code](https://github.com/osTicket/osTicket)) `GPL-2.0` `PHP`
 - [Pachno](https://pach.no/) - Bring your team together to design, build and deliver your project with a tool that works with you and your team, and adapts when you need to. ([Source Code](https://github.com/pachno/pachno)) `MPL-2.0` `PHP`
 - [Request Tracker](https://www.bestpractical.com/rt/) - An enterprise-grade issue tracking system. ([Source Code](https://github.com/bestpractical/rt)) `GPL-2.0` `Perl`
+- [Roundup Issue Tracker](https://www.roundup-tracker.org/) - A simple-to-use and -install issue-tracking system with command-line, web, REST, XML-RPC, and e-mail interfaces. Designed with flexibility in mind - not just another bug tracker. Unpack source, run “python demo.py”, open the printed demo URL!  ([Source Code](https://www.roundup-tracker.org/code.html)) `MIT, ZPL-2.0` `Python3 or Python2`
```

Comment:
please use the official description from https://www.roundup-tracker.org/

Reply:
- Please change the license tag to `MIT/ZPL-2.0`
- Please change the language tag to `Python`

Reply:
> please use the official description from https://www.roundup-tracker.org/

My description is almost word for word pulled from the site. The relevant
paragraphs are:

paragraph 1
```
Roundup is a simple-to-use and -install issue-tracking system with command-line, web and e-mail interfaces. It is based on the winning design from Ka-Ping Yee in the Software Carpentry “Track” design competition.
```
I added/highlighted the REST interface (mentioned in paragraph 2 of the README and at <https://www.roundup-tracker.org/docs/rest.html>) and the older xmlrpc (<https://www.roundup-tracker.org/docs/xmlrpc.html>) interface in the leading line to save space.

Paragraph 6 says:
```
…and so on. It’s been designed with [flexibility](https://www.roundup-tracker.org/docs/customizing.html) in mind - it’s not just another bug tracker. Roundup ships with a demo tracker to play with - after you’ve unpacked the source, just run “python demo.py” and load up the URL it prints out!
```
which was edited for brevity and the link dropped.

I think these are the important points for a short description and similar to other descriptions and announcements I have done. It:

* describes the software and how you can interact with it
* hints at its additional capabilities for those interested
* describes how to get a taste of Roundup using demo mode

Can you be more specific as to your concerns about my wording of the short description?

Also note that I will change the first line of the web site to more closely match the wording here. Historically we didn't highlight the API interfaces.

Reply:
> Can you be more specific as to your concerns about my wording of the short description?

Adding installation instructions `Unpack source, run “python demo.py”, open the printed demo URL!` to the description is going a bit far :) And descriptions must end with a `.`

Removing just this sentence would be fine